packages/editor/src/components/post-featured-image/index.js

```diff
@@ -250,6 +297,12 @@ const applyWithSelect = withSelect( ( select ) => {
 	const { getMedia, getPostType } = select( coreStore );
 	const { getCurrentPostId, getEditedPostAttribute } = select( editorStore );
 	const featuredImageId = getEditedPostAttribute( 'featured_media' );
+	const isRequestingFeaturedImageMedia =
+		!! featuredImageId &&
+		! select( coreStore ).hasFinishedResolution( 'getMedia', [
```

Comment:
We're already extracting the `coreStore` selectors above. Let's extract `hasFinishedResolution` there as well.